filip/models/ngsi_v2/subscriptions.py

```diff
@@ -296,6 +302,20 @@ def check_attrs(cls, v):
         else:
             raise TypeError()
 
+    @field_validator('alterationTypes')
+    def check_alteration_types(cls, v):
+        allowed_types = {"entityCreate", "entityDelete", "entityUpdate", "entityChange"}
+
+        if v is None:
+            return None
+        elif isinstance(v, list):
+            for item in v:
+                if item not in allowed_types:
+                    raise ValueError(f'{item} is not a valid alterationType')
```

Comment:
It would be super helpful if this error message also list the allowed alterationTypes.

Reply:
Sure. I have implemented it